### ISSUE TITLE
server level system

### ISSUE THREAD

**Issue by Rexogamer** [Rexogamer/cytrus-re#64]
Maybe with server level roles?

**Comment by Oxygemo** [comment]
Nah, IMO. Can we try to become less ordinary? I mean ofcourse we can have bans and things, but leveling system is stupid. @CommandBlock6417 @Rexogamer opinions?

**Comment by Rexogamer** [comment]
"Less ordinary" lotta bots don't have this feature (and annoyingly bots like mee6 put it behind a bloody paywall)
